add-ons/grafana/README.md

```diff
@@ -0,0 +1,41 @@
+# Grafana Add-on
+An addon that will deploy grafana with hawkular and prometheus datasources.
+It requires to connect to hawkular-metrics, provided by management-infra addon, and metrics should be enabled with --metrics on start up.
```

Comment:
Could you rewrite above line as `Prerequisites` to make it more visible?

## Prerequisites
- Start minishift with `--metrics` option and memory 4 GB (Not sure how much required but default memory 2GB might not work accurately here. I tried with 6GB though to be on safe side)
- Run 'management-infra' addon as:
```
$ minishift addon install management-infra
$ minishift addon apply management-infra
```